packages/zapp/console/src/components/Executions/Tables/NodeExecutionActions.tsx

```diff
@@ -0,0 +1,90 @@
+import { IconButton, Tooltip } from '@material-ui/core';
+import { NodeExecution } from 'models/Execution/types';
+import * as React from 'react';
+import InputsAndOutputsIcon from '@material-ui/icons/Tv';
+import { RerunIcon } from 'components/common/Icons/RerunIcon';
+import { Identifier, ResourceIdentifier } from 'models/Common/types';
+import { LaunchFormDialog } from 'components/Launch/LaunchForm/LaunchFormDialog';
+import { useNodeExecutionData } from 'components/hooks/useNodeExecution';
+import { TaskInitialLaunchParameters } from 'components/Launch/LaunchForm/types';
+import { literalsToLiteralValueMap } from 'components/Launch/LaunchForm/utils';
+import { NodeExecutionsTableState } from './types';
+import { useNodeExecutionContext } from '../contextProvider/NodeExecutionDetails';
+import { NodeExecutionDetails } from '../types';
+import t from './strings';
+
+export const NodeExecutionActions: React.FC<{
+  className?: string;
```

Comment:
Looks like className is never used, even if provided. can you please remove it from here?